File: Wave_V1/app/src/main/java/com/narsha/wave_android/view/fragment/search/searched/MusicSearchedFragment.java
package com.narsha.wave_android.view.fragment.search.searched;

import android.os.Bundle;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;
import androidx.recyclerview.widget.LinearLayoutManager;
import androidx.recyclerview.widget.RecyclerView;

import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.narsha.wave_android.R;
import com.narsha.wave_android.data.searched.SearchedData;
import com.narsha.wave_android.data.viewtype.SearchedViewType;
import com.narsha.wave_android.view.adapter.search.SearchedAllAdapter;

import java.util.ArrayList;

public class MusicSearchedFragment extends Fragment {
    ArrayList<SearchedData> arraySerached = new ArrayList<>();
    public MusicSearchedFragment() {
        // Required empty public constructor
    }


    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        // Inflate the layout for this fragment
        return inflater.inflate(R.layout.fragment_music_searched, container, false);
    }

    @Override
    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);

        arraySerached.add(new SearchedData("사랑이란 멜로는 없어", null, null, SearchedViewType.ViewType.MUSIC));
        arraySerached.add(new SearchedData("그대 행복해야 해요", null, null, SearchedViewType.ViewType.MUSIC));
        arraySerached.add(new SearchedData("사랑은 지날수록 더욱 선명하게 남아", null, null, SearchedViewType.ViewType.MUSIC));
        arraySerached.add(new SearchedData("어디에도", null, null, SearchedViewType.ViewType.MUSIC));
        arraySerached.add(new SearchedData("사계", null, null, SearchedViewType.ViewType.MUSIC));
        arraySerached.add(new SearchedData("넘쳐흘러", null, null, SearchedViewType.ViewType.MUSIC));
        arraySerached.add(new SearchedData("야생화", null, null, SearchedViewType.ViewType.MUSIC));
        arraySerached.add(new SearchedData("오래된 노래", null, null, SearchedViewType.ViewType.MUSIC));
        arraySerached.add(new SearchedData("너를 만나", null, null, SearchedViewType.ViewType.MUSIC));



        RecyclerView recyclerView = view.findViewById(R.id.recyclerSearchedMusic);

        LinearLayoutManager manager
                = new LinearLayoutManager(requireActivity(), LinearLayoutManager.VERTICAL,false);

        recyclerView.setLayoutManager(manager); // LayoutManager 등록
        recyclerView.setAdapter(new SearchedAllAdapter(arraySerached));  // Adapter 등록
    }
}

File: Wave_V2/app/src/main/java/com/project/wave_v2/view/fragment/searched/searched/MusicSearchedFragment.kt
package com.project.wave_v2.view.fragment.searched.searched

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import androidx.lifecycle.ViewModelProvider
import androidx.recyclerview.widget.LinearLayoutManager
import androidx.recyclerview.widget.RecyclerView
import com.narsha.wave_android.view.adapter.search.SearchedAllAdapter
import com.project.wave_v2.R
import com.project.wave_v2.data.response.search.SearchModel
import com.project.wave_v2.view.viewmodel.SearchedViewModel

class MusicSearchedFragment : Fragment() {

    var viewModel : SearchedViewModel = SearchedViewModel()
    var searchModel : SearchModel?= null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
    }

    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        // Inflate the layout for this fragment
        return inflater.inflate(R.layout.fragment_music_searched, container, false)
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)

        viewModel = ViewModelProvider(requireActivity()).get(SearchedViewModel::class.java)

        searchModel = viewModel.searchModel!!.value

        val recyclerView: RecyclerView = view.findViewById(R.id.recyclerSearchedMusic)
        val manager = LinearLayoutManager(requireActivity(), LinearLayoutManager.VERTICAL, false)
        recyclerView.layoutManager = manager // LayoutManager 등록
        recyclerView.adapter = SearchedAllAdapter(requireContext(), searchModel) // Adapter 등록
    }
}